ui/packages/catalog-ui-search/src/main/webapp/component/query-advanced/query-advanced.view.js

```diff
@@ -135,6 +135,9 @@ module.exports = Marionette.LayoutView.extend({
     }
   },
   edit: function() {
+    if (this.model.attributes.comparator === 'IS EMPTY') {
+      $('.if-editing').toggle()
```

Comment:
We should explicitly add a class that makes this invisible instead of calling `toggle()`